kaldb/src/main/java/com/slack/kaldb/logstore/LuceneIndexStoreImpl.java

```diff
@@ -203,7 +203,7 @@ public void addMessage(LogMessage message) {
     } catch (IOException e) {
       // TODO: For now crash the program on IOException since it is likely a serious issue.
       // In future may need to handle this case more gracefully.
-      e.printStackTrace();
+      LOG.error("failed to add document", e);
```

Comment:
Per the comment this should actually crash the program correct? Do we want to do a `new RuntimeHalterImpl().handleFatal(e);` as we do in the KaldbIndexer.java class?